cocos/core/geometry/aabb.ts

```diff
@@ -59,14 +57,20 @@ export class AABB {
     /**
       * @en
       * create a new AABB
+      * @param px  The x coordinate of the origin.
+      * @param py  The y coordinate of the origin.
+      * @param pz  The z coordinate of the origin.
+      * @param hw  Half the width of the AABB.
+      * @param hh  Half the height of the AABB.
+      * @param hl  Half the length of the AABB.
```

Comment:
The tool doesn't support this style. Should add both English comment and Chinese comment in the same line.